services/metadata/src/metadata/routes/domain/template.clj

```diff
@@ -105,3 +105,8 @@
 
    :name
    (describe String "The metadata template name")})
+
+(s/defschema CSVDownloadQueryParams
+  (assoc StandardUserQueryParams
```

Comment:
Yep, we've been keeping the user parameter on pretty much all of our endpoints, even if only for this reason.